admin/src/main/java/io/strimzi/arguments/topic/DescribeTopicCommand.java

```diff
@@ -29,13 +36,23 @@ public Integer call() {
      */
     private Integer describeTopics() {
         try (Admin admin = Admin.create(AdminProperties.adminProperties(this.bootstrapServer))) {
-            admin.describeTopics(List.of(this.getTopicPrefixOrName())).topicNameValues().forEach((key, value) -> {
+            List<KafkaTopicDescription> kafkaTopicDescriptionList = new LinkedList<>();
+
+            // parse response from java admin
+            admin.describeTopics(this.getListOfTopicNames()).topicNameValues().forEach((key, value) -> {
                 try {
-                    System.out.println(value.get());
+                    kafkaTopicDescriptionList.add(KafkaTopicDescription.parseKafkaTopicDescription(value.get()));
                 } catch (Exception e) {
-                    throw new RuntimeException("Unable to describe topic: " + this.getTopicPrefixOrName() + " due: " + e.getCause());
+                    throw new RuntimeException("Unable to describe topic: " + key + " due: " + e.getCause());
                 }
             });
+
+            // provide response
+            final String cmdOutput = DescribeTopicsUtils.getOutput(outputFormat, kafkaTopicDescriptionList);
+            System.out.println(cmdOutput);
+
```

Comment:
```suggestion
            // provide response
            System.out.println(DescribeTopicsUtils.getOutput(outputFormat, kafkaTopicDescriptionList));
```
I guess it's not needed to store the output in the variable and then use it just for printing it + unnecessary empty line